docs/source/io_formats/geometry.rst

```diff
@@ -370,3 +370,44 @@ Here is an example of a properly defined 2d hexagonal lattice:
                   202
         </universes>
     </hex_lattice>
+
+
+.. _dagmc_element:
+
+----------------------------
+``<dagmc_universe>`` Element
+----------------------------
+
+Each ``<dagmc_universe>`` element can have the following attributes or sub-elements:
+
+  :id:
+    A unique integer used to identify the universe.
+
+    *Default*: ""
+
+  :name:
+    An optional string name to identify the surface in summary output
+    files. This string is limited to 52 characters for formatting purposes.
```

Comment:
As we discussed offline, this limit is probably historical and no longer applies (unless you've found something that indicates we are indeed limited to 52 characters)

Reply:
Nope! I'll go ahead and update other instances of this as well.